### PR TITLE
fix: should be kib version not ib version

### PR DESCRIPTION
**Pull request by msdolbey**
**What problem does this PR solve?**:
typo for kib version (ib_version rather than kib_version)

**Which issue(s) does this PR fix?**:
N/A

**Special notes for your reviewer**:
typo in KIB Version

**Does this PR introduce a user-facing change?**:
No new features

### CODE REVIEW

**Review by mkarroqe — APPROVED** [review]
nice catch!

File: pkg/packer/manifests/vsphere/base/packer-base.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "export_manifest": "none",
         "folder": "",
         "guest_os_type": null,
-        "ib_version": "{{env `IB_VERSION`}}",
+        "kib_version": "{{env `KIB_VERSION`}}",
         "username": "{{env `VSPHERE_USERNAME`}}",
```
Comment (supershal) [inline pkg/packer/manifests/vsphere/base/packer-base.json:13]: building base templates from kib is deprecated and moved to https://github.com/mesosphere/vsphere-base-template .
I will file separate PR to remove these files.

File: pkg/packer/manifests/vsphere/packer.pkr.hcl [hunk 2]
```diff
@@ -104,9 +104,9 @@ variable "guest_os_type" {
   type = string
 }
 
-variable "ib_version" {
+variable "kib_version" {
```
Comment (supershal) [inline pkg/packer/manifests/vsphere/packer.pkr.hcl:107]: we are not using `KIB_VERSION` at all. we should remove it and its references from this file.

**Comment by msdolbey** [comment]
Closing - from Shalin:
"you can close it. I will file another PR right away. ..."

**Pull request closed without merging**